src/rgw/const.py

```diff
@@ -83,12 +83,14 @@
                       MOTR_MAX_RPC_MSG_SIZE_KEY, MOTR_RECONNECT_INTERVAL_KEY,
                       MOTR_RECONNECT_RETRY_COUNT_KEY]
 
+
 class RgwEndpoint(Enum):
     """Enum class to define rgw endpoints provided by hare."""
 
-    MOTR_PROFILE_FID = 'motr profile fid'
-    MOTR_HA_EP       = 'motr ha endpoint'
-    MOTR_CLIENT_EP   = 'motr my endpoint'
-    MOTR_PROCESS_FID = 'motr my fid'
+    MOTR_PROFILE_FID = {'profile_fid': 'motr profile fid'}
```

Comment:
Need to keep it as 
PROFILE_FID = {'profile_fid': 'motr profile fid'}

Reply:
addressed